From 533f40e097b8d08e1dfeb03e2484a4369fad778f Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@gmail.com>
Date: Wed, 8 Jun 2016 13:13:36 +0200
Subject: [PATCH] [FIX] apps: include stdarg.h in eventlogstring.h

va_list type is used in eventlogstring.h so it must include stdarg.h to define
it.

Upstream status: Pending
https://github.com/OpenAutomationTechnologies/openPOWERLINK_V2/pull/127

Fixes:
http://autobuild.buildroot.net/results/a9e/a9e7615a19922706039bf97ccb94bcf5b99330b2/build-end.log

Signed-off-by: Romain Naour <romain.naour@gmail.com>
---
 apps/common/src/eventlog/eventlog.c       | 1 -
 apps/common/src/eventlog/eventlogstring.c | 1 -
 apps/common/src/eventlog/eventlogstring.h | 2 ++
 3 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/apps/common/src/eventlog/eventlog.c b/apps/common/src/eventlog/eventlog.c
index f2193e2..eba1101 100644
--- a/apps/common/src/eventlog/eventlog.c
+++ b/apps/common/src/eventlog/eventlog.c
@@ -47,7 +47,6 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //------------------------------------------------------------------------------
 
 #include <stdio.h>
-#include <stdarg.h>
 #include <time.h>
 
 #include <oplk/debugstr.h>
diff --git a/apps/common/src/eventlog/eventlogstring.c b/apps/common/src/eventlog/eventlogstring.c
index 9fb677b..a55a1a4 100644
--- a/apps/common/src/eventlog/eventlogstring.c
+++ b/apps/common/src/eventlog/eventlogstring.c
@@ -47,7 +47,6 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "eventlogstring.h"
 
 #include <stdio.h>
-#include <stdarg.h>
 #include <time.h>
 
 #include <oplk/debugstr.h>
diff --git a/apps/common/src/eventlog/eventlogstring.h b/apps/common/src/eventlog/eventlogstring.h
index 2c291a6..3a5f2fa 100644
--- a/apps/common/src/eventlog/eventlogstring.h
+++ b/apps/common/src/eventlog/eventlogstring.h
@@ -41,6 +41,8 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //------------------------------------------------------------------------------
 // includes
 //------------------------------------------------------------------------------
+#include <stdarg.h>
+
 #include <oplk/oplk.h>
 #include <oplk/nmt.h>
 
-- 
2.5.5

